platform/security/pdp/security-pdp-authzrealm/pom.xml

```diff
@@ -111,8 +111,8 @@
                     <instructions>
                         <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
                         <Embed-Dependency>
-                            org.wso2.balana,
-                            commons-io
+                            arbitro-core,
+                            commons-io,
```

Comment:
I'm curious how this was working before. Neither `commons-io` nor `httpclient` would have been correctly embedded in the bundle, so either 
a) they never needed to be embedded because any classes we're getting from them are being imported or
b) there are code branches we not only don't test but that have never been exercised.